packages/typespec-test/test/widget_dpg/spec/main.tsp

```diff
@@ -112,8 +112,8 @@ It does not accept any options or parameters.
         @header nullableOptionalHeader?: string | null,
         @header bytesHeader: bytes,
         @header @encode(BytesKnownEncoding.base64) value: bytes,
-        @header({
-          format: "csv",
+        @header(#{
+          explode: true
```

Comment:
@MaryGao  what is the correct change for a csv/multi collection format here? I also see there's some cases in batch modular has changed this way. this is because they have $ sign in their name right?